invenio/modules/deposit/validation_utils.py

```diff
@@ -165,7 +165,7 @@ def __init__(self, message=None):
         self.regexp = re.compile(self.pattern, re.I)
         self.message = message if message else (
             "The provided DOI is invalid - it should look similar to "
-            "'10.1234/foo.bar'.")
```

Comment:
Thanks @lnielsen I will remove this commit and just pass the custom message from the overlay.